packages/core/src/components/sections/Newsletter/Newsletter.tsx

```diff
@@ -42,14 +52,22 @@ export interface NewsletterProps
    * The subscribe button label.
    */
   subscribeButtonLabel?: string
+  /**
+   * The subscribe button loading label.
+   */
+  subscribeButtonLoadingLabel?: string
   /**
    * The card Variant
    */
-  card: Boolean
+  card?: UINewsletterProps['card']
 
-  toastSubscribe: SubscribeMessage
+  toastSubscribe?: SubscribeMessage
 
-  toastSubscribeError: SubscribeMessage
+  toastSubscribeError?: SubscribeMessage
+  /**
+   * Specifies the component's color variant combination.
+   */
+  colorVariant?: UINewsletterProps['colorVariant']
```

Comment:
Just to improve the readability, can we reorder so UINewsletterProps can be listed together? 

privacyPolicy, card, colorVariant, subscribeButtonLabel...